employee.rb

```diff
@@ -0,0 +1,22 @@
+
+class Employee
+  attr_accessor :name, :salary, :performance_review, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @performance_review = []
+    @satisfactory = []
+  end
+
+  def pay_raise(employee)
```

Comment:
Ah, okay.  Since you're already in the employee class, you have access to the employee.  Instead, you're going to want to pass in the raise_percent as a parameter, which would be better than setting it to always be 7%.  

Out of curiosity, why did you pick 7%?